dinit-service/polkitd/files/polkitd

```diff
@@ -0,0 +1,6 @@
+type = process
```

Comment:
This file should not be named polkitd. We match the name of the package that provides the daemon (in this case `sys-auth/polkit`). The service file is correctly named though, take a look at `dinit-service/openssh` for an example.